### ISSUE TITLE
204 | No Content Responses Throwing Error

### ISSUE THREAD

**Issue by ASRagab** [swagger-mock/swagger-mock#17]
Thank you for your library, perhaps I am misusing it but I have seem to run into an inconsistency:

For a post endpoint, if I have a `204` No Content listed as a response for a POST request, I get some interesting behavior:

If I document it like this in the swagger:
```yaml
      responses:
        204:
          description: accepted - no content
```
When attempting to post to endpoint I receive an Internal Server Response 500:

```
Negotiation\Exception\InvalidArgument: A set of server priorities should be given. in /app/vendor/willdurand/negotiation/src/Negotiation/AbstractNegotiator.php:19
Stack trace:
#0 /app/src/Mock/Negotiation/MediaTypeNegotiator.php(73): Negotiation\AbstractNegotiator->getBest('*/*', Array)
#1 /app/src/Mock/Negotiation/MediaTypeNegotiator.php(45): App\Mock\Negotiation\MediaTypeNegotiator->getBestMediaType('*/*', Array)
#2 /app/src/Mock/MockResponseGenerator.php(55): App\Mock\Negotiation\MediaTypeNegotiator->negotiateMediaType(Object(Symfony\Component\HttpFoundation\Request), Object(App\Mock\Parameters\MockResponse))
#3 /app/src/API/RequestHandler.php(43): App\Mock\MockResponseGenerator->generateResponse(Object(Symfony\Component\HttpFoundation\Request), Object(App\Mock\Parameters\Endpoint))
#4 /app/src/EventListener/RequestListener.php(32): App\API\RequestHandler->handleRequest(Object(Symfony\Component\HttpFoundation\Request))
#5 /app/vendor/symfony/event-dispatcher/EventDispatcher.php(298): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#6 /app/vendor/symfony/event-dispatcher/EventDispatcher.php(260): Symfony\Component\EventDispatcher\EventDispatcher::Symfony\Component\EventDispatcher\{closure}(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#7 /app/vendor/symfony/event-dispatcher/EventDispatcher.php(235): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#8 /app/vendor/symfony/event-dispatcher/EventDispatcher.php(75): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#9 /app/vendor/symfony/http-kernel/HttpKernel.php(127): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#10 /app/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#11 /app/vendor/symfony/http-kernel/Kernel.php(198): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#12 /app/bin/worker.php(48): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
```

However, if I document the response like this, i.e. including a valid `content`:

```yaml
     responses:
        204:
          description: accepted - no content
          content:
            application/json:
              schema:
                properties:
                  verified:
                    type: boolean
                    example: true
                required:
                  - verified
```

I get back the correct empty 204 No Content response, I think it is looking for a `content` to parse.

Thank you for looking into this.

**Comment by strider2038** [comment]
@ASRagab than you for report! Issue fixed in [v0.1.8](https://github.com/swagger-mock/swagger-mock/releases/tag/v0.1.8)

**Comment by ASRagab** [comment]
> @ASRagab than you for report! Issue fixed in [v0.1.8](https://github.com/swagger-mock/swagger-mock/releases/tag/v0.1.8)

Worked like charm!
